conf/agent/agent_full.conf

```diff
@@ -352,25 +352,28 @@ plugins {
             # node_name: The name of the node. Overrides the value obtained by
             # the environment variable specified by node_name_env.
             # node_name = ""
-
-            sigstore {
-                # rekor_url: The URL for the rekor STL Server to use with cosign.
-                # rekor_url = "https://rekor.sigstore.dev"
-                
-                # skip_signature_verification_image_list: List of images that should
-                # not be verified by cosign. They will receive a default
-                # sigstore-validation:passed selector, but no other sigstore related selectors.
-                # skip_signature_verification_image_list = ["sha:image1hash","sha:image2hash"]
-
-                # enable_allowed_subjects_list: Boolean indicating whether image 
-                # signatures will be checked against a list of subjects.
-                # enable_allowed_subjects_list = false
-
-                # allowed_subjects_list: List of subjects that image signatures
-                # will be checked against, if enabled through the above option.
-                # signatures from subjects outside this list will receive
-                # no sigstore-related selectors. These should be email addresses.
-                # allowed_subjects_list = ["subject1@example.com","subject2@example.com"]
+            experimental {
+                # enable_sigstore = false
```

Comment:
I think we could to use the `check_signature_enabled` flag instead of creating one more